test/test_01_hw_06.py

```diff
@@ -0,0 +1,32 @@
+from hw.hw_06_task_01 import instances_counter
+
+
+@instances_counter
+class User:
+    pass
+
+
+def test_count_before_creation_is_zero():
+    count = User.get_created_instances()
+    assert count == 0
+
+
+def test_count_of_created_instances():
+    user, _, _ = User(), User(), User()  # noqa
+    count = User.get_created_instances()
+    assert count == 3
+
+
+@instances_counter
+class FirstClass:
+    def __init__(self, number: int):
+        self.counter = number
```

Comment:
Sorry missed that, please create a class atribute counter and test it.

```
class FirstClass:
   counter = 6
```

Reply:
It's obvious that we will have a conflict, because both counters are class attributes. Can I just rename cls.counter to cls.counter_ ? Otherwise I need a clue...

Reply:
> Can I just rename cls.counter to cls.counter_ ?

If you do it, guess my next question :)

> Otherwise I need a clue...

Here it is: https://github.com/cdeler/epam_python_autumn_2020/tree/main/lecture_06_oop2_exceptions/slides